forge-gui/res/cardsfolder/upcoming/fires_of_mount_doom.txt

```diff
@@ -0,0 +1,14 @@
+Name:Fires of Mount Doom
+ManaCost:2 R
+Types:Legendary Enchantment
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDamage | TriggerDescription$ When CARDNAME enters the battlefield, it deals 2 damage to target creature an opponent controls. Destroy all Equipment attached to that creature.
+SVar:TrigDamage:DB$ DealDamage | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select target creature an opponent controls | NumDmg$ 2 | SubAbility$ ChompEquip
+SVar:ChompEquip:DB$ DestroyAll | ValidCards$ Targeted.Equipment+Attached
+A:AB$ Dig | Cost$ 2 R | DigNum$ 1 | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect | SpellDescription$ Exile the top card of your library. You may play that card this turn. When you play a card this way, CARDNAME deals 2 damage to each player.
+SVar:DBEffect:DB$ Effect | StaticAbilities$ STPlay | Triggers$ TriggerCastDoM,TriggerLandPlayed | ForgetOnMoved$ Exile | ForgetOnCast$ False | RememberObjects$ Remembered | SubAbility$ DBCleanup

```

Comment:
does it trigger when playing lands? the timing might be tricky

Reply:
I think so  
![image](https://github.com/Card-Forge/forge/assets/67333662/597bd9cb-5cb2-40ea-9991-48707399ca9b)

Reply:
I think so 
![image](https://github.com/Card-Forge/forge/assets/67333662/d4503b66-d2d5-4e5f-b5c5-0ff501278c36)